integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/dataload/TestLoadDataWithHiveSyntax.scala

```diff
@@ -76,6 +77,35 @@ class TestLoadDataWithHiveSyntax extends QueryTest with BeforeAndAfterAll {
 
   }
 
+  test("create table with smallint type and query smallint table")({
+    sql(
+      "create table smallinttable(empno smallint, empname String, designation string, " +
```

Comment:
@cenyuhai Please create carbontable instead of hive table.Use `STORED BY 'org.apache.carbondata.format'`  while creating table to create carbon table.